### ISSUE TITLE
Feature request: SOCKS5 UDP support

### ISSUE THREAD

**Issue by ulinuxp** [wongsyrone/transocks-wong#5]
Would you consider supporting UDP over SOCKS5 with iptables TPROXY ? It would be great for DNS queries and gaming.

**Comment by phantomcraft** [comment]
I don't want to make spam, but there is a project which supports UDP and TPROXY, called ipt2socks:

https://github.com/zfl9/ipt2socks

As it uses libev, should support splice syscall.

**Comment by ulinuxp** [comment]
@phantomcraft have you tried ipt2socks before? I have no luck trying that.. it just keeps crashing

**Comment by phantomcraft** [comment]
@ulinuxp Yes, I tested it but only the "redirect" scheme, not the "tproxy". I really don't know how to forward UDP to a subnet with "tproxy", with "redirect" I just use:

iptables -t nat -A PREROUTING -i subnet0 -p tcp --syn -j DNAT --to-destination <some_ip>:<some_port>

Worked well and seems to use splice syscall as the speed reached ~30gbps in my system.
